typings/internal-electron.d.ts

```diff
@@ -16,6 +16,7 @@ declare namespace Electron {
     setVersion(version: string): void;
     setDesktopName(name: string): void;
     setAppPath(path: string | null): void;
+    getAppPath(): string;
```

Comment:
this is already in electron.d.ts via https://github.com/electron/electron/blob/main/docs/api/app.md#appgetapppath

Reply:
@miniak Thank you very much! I didn't notice that the `electron.d.ts` file got generated with the needed types